intro-HPC/ch_apptainer.tex

```diff
@@ -31,34 +31,35 @@ \section{Restrictions on image location}
 
 \section{Available filesystems}
 
-All HPC-UGent shared filesystems will be readily available in a Singularity container,
+All HPC-UGent shared filesystems will be readily available in an Apptainer/Singularity container,
 including the home, data and scratch filesystems, and they will be accessible via the
 familiar \lstinline|$VSC_HOME|, \lstinline|$VSC_DATA*| and \lstinline|$VSC_SCRATCH*| environment variables.
 
 \section{Singularity Images}
```

Comment:
@hajgato This should also say `Apptainer/Singularity` ? Same for `subsection` below

Reply:
I was hesitating as apptainer still calls the images (.sif or .simg) to singularity containers. So I am not sure whether we have to stuck to Apptainer/Singularity image(s) or Singularity image(s). I decided to use Apptainer/Singularity images to avoid some confusion. For sure the mixed solution is probably not the best. @boegel thoughts?

Reply:
fixed